README.md

```diff
@@ -1,2 +1,16 @@
-# terminology
-Codesystems, questionnaires, and validation tools for them, for the All of Us project.
+# All of Us Terminology
+
+Codesystems, questionnaires, and validation tools for them, for the All of Us
+project.
+
+## Usage
+
+![workflow diagram](doc/overview.png)
+
+1.  Export questionnaires from Vibrent's authoring tool and place them in
+    `source/`.
+1.  `tools/validate_and_publish [--commit]` writes to `generated/`.
```

Comment:
(Or, if "commit" includes push, we should document it that way. My preference would be to leave the push as an explicit step, but if `--commit` is intended to push too, we should call it something more like `--commit-and-push`)